tests/Jaeger/Codec/B3CodecTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Jaeger\Codec;
+
+use const Jaeger\DEBUG_FLAG;
+use const Jaeger\SAMPLED_FLAG;
+use Jaeger\SpanContext;
+use PHPUnit\Framework\TestCase;
+
+class B3CodecTest extends TestCase
+{
+    /** @var B3Codec */
+    private $codec;
+
+    function setUp()
+    {
+        $this->codec = new B3Codec;
+    }
+
+    function testInject()
+    {
+        // Given
+        $traceId = 123;
+        $spanId = 456;
+        $parentId = 789;
+
+        $spanContext = new SpanContext(
+            $traceId,
+            $spanId,
+            $parentId,
+            SAMPLED_FLAG
+        );
+        $carrier = array();
+
+        // When
+        $this->codec->inject($spanContext, $carrier);
+
+        // Then
+        $this->assertEquals('7b', $carrier['X-B3-TraceId']);
+        $this->assertEquals('1c8', $carrier['X-B3-SpanId']);
+        $this->assertEquals('315', $carrier['X-B3-ParentSpanId']);
+        $this->assertEquals('1', $carrier['X-B3-Flags']);
+    }
+
+    function testExtract()
```

Comment:
You should add an extract test without the parent span and with invalid hex value for the header